windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -175,12 +175,13 @@ cp $LIB_GCC_DLL $SD_LOGGING_AGENT_DIR_BIN
 #  STEP 5.1 - TEMPORARY HACK TO UPDATE RUBY FILE
 ##############################
 # 
-# TODO: Remove when https://github.com/djberg96/win32-eventlog/pull/23 is merged.
+# TODO: Remove when https://github.com/djberg96/win32-eventlog/pull/23 is merged and released.
```

Comment:
Should these references to pull requests be in the replacement text as well? It would help associate which line of the replacement comes from which request...